proxy/InkAPI.cc

```diff
@@ -8153,6 +8153,10 @@ _conf_to_memberp(TSOverridableConfigKey conf, OverridableHttpConfigParams *overr
     typ = OVERRIDABLE_TYPE_STRING;
     ret = &overridableHttpConfig->client_cert_filepath;
     break;
+  case TS_CONFIG_PARENT_FAILURES_UPDATE_HOSTDB:
+    typ = OVERRIDABLE_TYPE_INT;
```

Comment:
Now that I see this, this type should probably be 'byte'. See some of the earlier examples how to do that (both in the SM structs as well as here). OVERRiDABLE_TYPE_BYTE is also the default, and we should use that (I probably need to take a look at this, to make sure it's done correctly elsewhere as well).

Also, I think the way this was written, the string types were all collected towards the end. Not sure that's important any more, but might be worth keeping that semantics?